src/components/dashboard/Claim.js

```diff
@@ -50,7 +50,13 @@ const log = logger.child({ from: 'Claim' })
 
 const LoadingAnimation = ({ success, speed = 3 }) => (
   <View style={{ alignItems: 'center' }}>
-    <SpinnerCheckMark successSpeed={speed} success={success} width={175} height={'auto'} />
+    <SpinnerCheckMark
+      successSpeed={speed}
+      success={success}
+      width={145}
+      height={Platform.select({ web: 'auto', default: 145 })}
```

Comment:
move those calculations inside  SpinnerCheckMark - they should be applied everyhere we using that component

Reply:
@serdiukov-o-nordwhale SpinnerCheckMark already has defaults. What I can do is add in SpinnerCheckMark a check to see if the incoming height/width are int or a different type, what do you think?

Reply:
- width should have default value in the props
- height / marginTop shouldn't have defaults
- if  height / marginTop are set, the values passed are used
- otherwise height / marginTop are calculated according to that formulas (e.g. Platform.select({ web: 'auto', default: width }) etc)

Reply:
fixed in last commit